docs/CUSTOM_RULES.md

```diff
@@ -11,22 +10,25 @@ rules that rely on the API provided by SonarDelphi.
 
 ## Content
 
-* [Getting Started](#getting-started)
-    * [Looking at the pom](#looking-at-the-pom)
-* [Writing a rule](#writing-a-rule)
-    * [Two files to forge a rule](#two-files-to-forge-a-rule)
-    * [A specification to make it right](#a-specification-to-make-it-right)
-    * [A test class to rule them all](#a-test-class-to-rule-them-all)
-    * [First version: Using AST Nodes and API basics](#first-version-using-ast-nodes-and-api-basics)
-    * [Second version: Using semantic API](#second-version-using-semantic-api)
-    * [What you can use, and what you can't](#what-you-can-use-and-what-you-cant)
-* [Registering the rule in the custom plugin](#registering-the-rule-in-the-custom-plugin)
-    * [Rule Metadata](#rule-metadata)
-    * [Rule Activation](#rule-activation)
-    * [Rule Registrar](#rule-registrar)
-* [Testing a custom plugin](#testing-a-custom-plugin)
-    * [How to define rule parameters](#how-to-define-rule-parameters)
-* [References](#references)
+- [Writing Custom Delphi Rules](#writing-custom-delphi-rules)
+  - [Content](#content)
```

Comment:
Does it make sense to put the table of contents into the table of contents?
Seems a bit weird to me.

Reply:
This table of contents is auto-generated and maintained by the [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) in VSCode 😅 I can remove this manually if it bothers you.

Reply:
Removed - I can exclude it from the generation.